routes/users.js

```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const router = express.Router();
+const createError = require("http-errors");
+const admin = require("firebase-admin");
+
+const User = require("../models/User");
+
+const MESSAGE = require("../constants/messages");
+const ERROR = require("../constants/error");
+
+router.get("/check_member", async (req, res, next) => {
+  const { EXPIRED_TOKEN, UNEXPECTED_ERROR, OK } = MESSAGE;
+  const { AUTH_ID_TOKEN_REVOKED, AUTH_ID_TOKEN_EXPIRED } = ERROR;
+  const { idToken } = req.body;
+
+  try {
+    const decodeToken = await admin
+      .auth()
+      .verifyIdToken(idToken, false);
+
+    const { email } = decodeToken;
+
+    const user = await User.findOne({ email });
+
+    const hasUserData = user === null ? false : true;
```

Comment:
boolean 값을 반환하는 삼항 연산자의 사용은 지양하는 것이 좋습니다.

대신 다음과 같이 작성할 수 있습니다!

```suggestion
    const hasUserData = user !== null;
```